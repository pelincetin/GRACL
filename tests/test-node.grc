int main (){
    Node n = createNode("yo");
    printNode(n);
    return 0;
}
