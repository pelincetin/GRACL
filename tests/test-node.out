Hello
Not visited
zac efron
Not visited
Not Visited
Visited
Hello
Not visited
Not Equal
Equal
